### DISCUSSION TITLE
Should I translate sidebar?

### DISCUSSION THREAD

**Discussion by tamagoez** [saadeghi/daisyui#609] [Q&A] [answered]
I found Chinese sidebar translated like `kbd (kbd in China)`.
Shoud I translate Japanese sidebar like this?

**Comment by saadeghi** [comment, marked as answer]
It's the HTML [tag name](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/kbd) so I think `kbd` is okay but If you think it needs a description in parentheses, that's good too.
